README.md

```diff
@@ -53,17 +53,40 @@ The examples below demonstrate how you would stage Synapse files in an S3 bucket
     foobar,s3://example-scratch/synstage/syn28521174/foobar.R1.fastq.gz,s3://example-scratch/synstage/syn28521175/foobar.R2.fastq.gz,unstranded
     ```
 
+### Special Considerations for Seven Bridges CGC Files
+
+If you are staging Seven Bridges files, there are a few differences that you will want to incorporate in your Nextflow run. 
+
+When adding your URIs to your input file, Seven Bridges file URIs should have the prefix `sbg://`. 
+
+There are two ways to get the ID of a file in Seven Bridges:
+
+1. The first way involves simply logging in to [Seven Bridges CGC](https://cgc-accounts.sbgenomics.com/auth/login), navigating to the file and copying the ID from the URL. For example, your URL might look like this: "https://cgc.sbgenomics.com/u/brmacdonald/invite-test/files/63b717559fd1ad5d228550a0/". From this url, you would copy the "63b717559fd1ad5d228550a0" piece and combine it with the `sbg://` prefix to have the complete URI `sbg://63b717559fd1ad5d228550a0`.
+2. The second way involves using the [SBG CLI](https://docs.sevenbridges.com/docs/files-and-metadata). To get the ID numbers that you need, run the `sb files list` command and specify the project that you are downloading files from. A list of all files in the project will be returned, and you will combine the ID number with the prefix for each file that you want to stage.
+
+`nf-synstage` can handle either or both types of URIs in one run.
+
+
 ## Authentication
 
+### Synapse 
+
 Downloading files from Synapse requires the workflow to be authenticated. The workflow currently supports two authentication methods:
 
 - **(Preferred)** Create a secret called `SYNAPSE_AUTH_TOKEN` containing a Synapse personal access token using the [Nextflow CLI](https://nextflow.io/docs/latest/secrets.html) or [Nextflow Tower](https://help.tower.nf/latest/secrets/overview/).
 - Provide a Synapse configuration file containing a personal access token (see example above) to the `synapse_config` parameter. This method is best used if Nextflow/Tower secrets aren't supported on your platform. **Important:** Make sure that your `synapse_config` file is not stored in a directory that will be indexed on or uploaded to Synapse.
 
-### Personal access tokens
+#### Personal access tokens
 
 You can generate a Synapse personal access token using [this dashboard](https://www.synapse.org/#!PersonalAccessTokens:).
 
+### Seven Bridges CGC
+
+To authenticate a Seven Bridges account, you need to configure two secrets. In order to retrieve your secrets, login to [Seven Bridges CGC](https://cgc-accounts.sbgenomics.com/auth/login), click on the "Developer" dropdown and click on "Authentication Token". 
+
+1. Copy your Authentication Token and create a secret called `SB_AUTH_TOKEN` using the [Nextflow CLI](https://nextflow.io/docs/latest/secrets.html) or [Nextflow Tower](https://help.tower.nf/latest/secrets/overview/).
+2. Copy the API endpoint and create a secret called `SB_API_ENDPOINT` using the same method.
```

Comment:
Could you add a link to [here](https://sevenbridges-python.readthedocs.io/en/latest/quickstart/#authentication-and-configuration), which lists the various API endpoints that someone can pick from?

Reply:
added